OsmAnd/res/values/strings.xml

```diff
@@ -41,29 +41,29 @@
     <string name="default_speed_setting_title">Default speed</string>
     <string name="default_speed_setting_descr">Change default speed settings</string>
     <string name="minmax_speed_dialog_title">Set min/max speed</string>
-    <string name="default_speed_dialog_msg">Used to calculate the expected arrival time during routing</string>
+    <string name="default_speed_dialog_msg">Used to calculate expected time of arrival during routing</string>
     <string name="new_profile">New profile</string>
     <string name="shared_string_crash">Crash</string>
-    <string name="last_launch_crashed">The last launch of OsmAnd failed. Please help us to improve OsmAnd by sharing the error message.</string>
-    <string name="press_again_to_change_the_map_orientation">Tap again to change the map orientation</string>
+    <string name="last_launch_crashed">OsmAnd failed to start last time. Please help improve OsmAnd by sharing the error message.</string>
+    <string name="press_again_to_change_the_map_orientation">Tap again to change map orientation</string>
     <string name="process_downloading_service">OsmAnd downloading service</string>
     <string name="shared_string_color_magenta">Magenta</string>
     <string name="shared_string_icon">Icon</string>
-    <string name="rate_dialog_descr">Please give us 30 seconds by sharing your feedback and rating our work on Google Play.</string>
+    <string name="rate_dialog_descr">Please share your feedback and rate our work on Google Play.</string>
     <string name="button_rate">Rate</string>
     <string name="shared_string_privacy_policy">Privacy Policy</string>
-    <string name="help_us_make_osmand_better">Help us make OsmAnd better!</string>
-    <string name="make_osmand_better_descr">Allow OsmAnd to collect and process anonymous application usage data. We do not collect or store data about your position, or about any locations you view on the map.\n\nYou can change your selection at any time in Settings > Privacy and Security.</string>
-    <string name="choose_data_to_share">Choose what type of data you want to share:</string>
+    <string name="help_us_make_osmand_better">Help make OsmAnd better!</string>
+    <string name="make_osmand_better_descr">Allow OsmAnd to collect and process anonymous app usage data. Your position is not collected nor stored, neither is any locations you view on the map.\n\nChange your choice any time in \'Settings\' → \'Privacy and Security\'.</string>
```

Comment:
The first part of this is very much a copy of "do_not_send_anonymous_app_usage_desc".